config/config_partial.go

```diff
@@ -105,11 +112,12 @@ type terragruntInputs struct {
 // be decoded even in partial decoding, because they provide bindings that are necessary for parsing any block in the
 // file. Currently base blocks are:
 // - locals
+// - features
 // - include
-func DecodeBaseBlocks(ctx *ParsingContext, file *hclparse.File, includeFromChild *IncludeConfig) (*TrackInclude, *cty.Value, error) {
+func DecodeBaseBlocks(ctx *ParsingContext, file *hclparse.File, includeFromChild *IncludeConfig) (*TrackInclude, *cty.Value, *cty.Value, error) {
```

Comment:
It's hard to understand what these return values are. Should we restructure this so that it returns one wrapper struct and an error?